modules/working-in-code-server.adoc

```diff
@@ -55,11 +55,12 @@ The *Data Science Projects* page opens.
 +
 A project details page opens.
 . Click the *Workbenches* tab.
-. If the workbench that you want to use is not already running, click the action menu (*&#8942;*) beside the workbench, and click *Start*.
+. If the status of the workbench that you want to use is *Running*, you can skip this step. 
```

Comment:
same comment as above > "skip to the next step" ("you can" makes it seem optional)